CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Communication Channels
 ----------------------
 
 Most communication about Bitcoin Core development happens on IRC, in the
-#bitcoin-core-dev channel on Freenode. The easiest way to participate on IRC is
+#bitcoin-core-dev channel on Freenode. The easiest way to participate in IRC is
```

Comment:
This is incorrect.

Reply:
Reverted back that particular change in the 2nd commit.